pkg/rulefmt/rulefmt.go

```diff
@@ -45,19 +46,23 @@ type RuleGroups struct {
 	Groups []RuleGroup `yaml:"groups"`
 }
 
+type ruleGroups struct {
+	Groups []yaml.Node `yaml:"groups"`
+}
+
 // Validate validates all rules in the rule groups.
 func (g *RuleGroups) Validate() (errs []error) {
 	set := map[string]struct{}{}
 
-	for _, g := range g.Groups {
+	for j, g := range g.Groups {
 		if g.Name == "" {
-			errs = append(errs, errors.Errorf("Groupname should not be empty"))
+			errs = append(errs, errors.Errorf("Groupname should not be empty, line: %d, column: %d", grpsTmp.Groups[j].Line, grpsTmp.Groups[j].Column))
```

Comment:
Do we have to add the line/col in every error?

Reply:
yes. At each error, we can find both the line and column number, which has been implemented here.